### PR TITLE
docs: Added installation instructions for gentoo (#556) [skip ci]

### PR DESCRIPTION
**Pull request by chriscpritchard**
#### Description
Added gentoo install instructions

#### Todos

N/A documentation only

#### Issues Fixed or Closed by this PR

None

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -105,11 +105,17 @@ yay -S overseer
 ```
 {% endtab %}
 
-{% endtabs %}
+{% tab title="Gentoo \(3rd Party\)" %}
+Portage overlay: [https://github.com/chriscpritchard/overseerr-overlay](https://github.com/chriscpritchard/overseerr-overlay)
 
+Efforts will be made to keep up to date with the latest releases, however, this cannot be guaranteed.
 
+To enable using eselect repository, run ```eselect repository add overseerr-overlay git https://github.com/chriscpritchard/overseerr-overlay.git```
```
Comment (flying-sausages) [inline docs/getting-started/installation.md:113]:
```suggestion
To enable using eselect repository, run `eselect repository add overseerr-overlay git https://github.com/chriscpritchard/overseerr-overlay.git`
```

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -105,11 +105,17 @@ yay -S overseer
...
 
+Once complete, you can just run ```emerge www-apps/overseerr```
```
Comment (flying-sausages) [inline docs/getting-started/installation.md:115]:
```suggestion
Once complete, you can just run `emerge www-apps/overseerr`
```

File: docs/getting-started/installation.md [hunk 3]
```diff
@@ -105,11 +105,17 @@ yay -S overseer
 ```
 {% endtab %}
 
-{% endtabs %}
+{% tab title="Gentoo \(3rd Party\)" %}
+Portage overlay: [https://github.com/chriscpritchard/overseerr-overlay](https://github.com/chriscpritchard/overseerr-overlay)
```
Comment (flying-sausages) [inline docs/getting-started/installation.md:109]:
```suggestion
[Portage overlay GitHub repository](https://github.com/chriscpritchard/overseerr-overlay)
```

**Review by flying-sausages — COMMENTED** [review]
some small formatting :)

**Review by flying-sausages — APPROVED** [review]

**Review by samwiseg0 — APPROVED** [review]
Thanks!

**Pull request merged**